js/index.js

```diff
@@ -25,4 +25,120 @@ menuIcon.addEventListener('click', openMenuModal);
 menuCloseBtn.addEventListener('click', closeMenuModal);
 listItems.forEach((item) => {
   item.addEventListener('click', closeMenuModal);
-});
\ No newline at end of file
+});
+
+/** *******************************************
+* Prject section JS implementation
+********************************************* */
+const worksSection = document.getElementById('works');
+const projectData = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-1.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post <br />Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-2.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-3.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 4,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-4.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+];
```

Comment:
- [ ] God job :+1: storing your projects in an array of objects but I think it would be better if you store four different projects instead of having one project respected thrice so that one can differentiate each project when clicking on the `see project` button to see a project. :relaxed: